dti-website/src/components/DtiProject.tsx

```diff
@@ -21,10 +21,10 @@ export default function DtiProject({
   const project = allProjects.find((it) => it.teamId === teamId);
   if (project == null) throw new Error();
   const pastMembers = allMembers.filter((member) =>
-    (member.otherSubteams || []).includes(teamId)
+    ((member.formerSubteams as string[]) || []).includes(teamId)
   );
-  const currentMembers = allMembers.filter(
-    (member) => member.subteam === teamId
+  const currentMembers = allMembers.filter((member) =>
+    member.subteams ? (member.subteams as string[]).includes(teamId) : false
```

Comment:
probably don't need cast here

Reply:
Getting `Argument of type 'string' is not assignable to parameter of type 'never'` without the cast